sbws/util/stem.py

```diff
@@ -72,9 +75,11 @@ def init_controller(port=None, path=None, set_custom_stream_settings=True):
 
 
 def is_bootstrapped(c):
-    if not is_controller_okay(c):
+    try:
+        line = c.get_info('status/bootstrap-phase')
+    except Exception as e:
+        log.exception("Exception bootstrapping %s", e)
```

Comment:
We aren't bootstrapping in this function ... we are checking to see if we are bootstrapped. I'd say "Exception trying to check bootstrap phase" or something to that effect.